src/transformers/modeling_outputs.py

```diff
@@ -245,6 +245,41 @@ class BaseModelOutputWithPastAndCrossAttentions(ModelOutput):
     cross_attentions: Optional[Tuple[torch.FloatTensor]] = None
 
 
+@dataclass
+class DecisionTransformerOutput(ModelOutput):
+    """
+    Base class for model's outputs that also contains a pooling of the last hidden states. TODO
```

Comment:
```suggestion
    Base class for model's outputs that also contains a pooling of the last hidden states.
```
That is left to do here?